DIRECTOR_HANDBOOK.md

```diff
@@ -0,0 +1,92 @@
+
+* __Attendance at Meetings__:  You should attend all meetings of the Board of Directors (except when special circumstances or pre-established material commitments dictate otherwise).  In order to do its job, the Board must act as a group, so your attendance at meetings is critical to ensure that all voices are heard.
+* __Exercising Independent Judgment__:  Directors serve as individuals, and the effectiveness of any vote or action they take as a member of the Board should not depend on whether it was authorized or directed by any other person or entity or was contrary to the instructions or interests of any other person or entity.  Rather, you should weigh your votes on official action carefully, based upon your independent business judgment, and you should abstain or dissent when you believe any action for which approval is sought is not in the best interests of the Foundation or its mission.
+* __Continuing to be Informed__:  In order for you to make prudent decisions for the Foundation, it is important that you remain informed.  In furtherance of this, you should read all materials supplied to you and, to the extent that the information provided is inadequate, you should request additional information.  In addition, you should review carefully the Foundation’s Certificate of Incorporation and By-Laws.
+* __Delegating Responsibilities__:  Since the Board of Directors does not perform the day-to-day operations of the Foundation, it must prudently select officers and managers and delegate these operations.  Once delegated, however, the Board must oversee the Foundation’s officers and managers to ensure they are acting responsibly.
+* __Acting in the Foundation’s Best Interest__:  You should always act in good faith and in a manner you reasonably believe to be in the Foundation’s best interest, and with independent and informed judgment.
+
+Your _duty of loyalty_ means, generally, that you exercise your powers as a Director in the interest of the Foundation and not in your own interests or the interests of another entity or person.  You may not use your position as a Director for personal advantage or to take an opportunity that is offered to the Foundation and use it personally.  In some cases, you may have interests that conflict with those of the Foundation.  You must be aware of the potential for such conflicts and disclose any potential conflict to the Board when this type of situation arises, and must otherwise comply with the provisions of Article X of the By-Laws and of any Conflicts of Interest Policy that is then in place.
+
+In addition, you should keep all matters and information relating to the Foundation in complete confidence until there has been general public disclosure by the appropriate spokesperson, or unless the information is already a matter of public record or common knowledge.  All Directors are required to use reasonable care to protect and safeguard the Foundation’s confidential information.
```

Comment:
Do we need to identify the level of confidentiality for each document to cover scenarios where information has not been disclosed to the “general public”, but requires sharing outside the Board?  For example, materials the Board shares with the Legal Committee, Marketing Committee, etc. (where non-Board members participate), materials that the Board wants to review with Foundation staff, the TSC, or other community members to solicit feedback, etc.